Orange/widgets/regression/owrandomforestregression.py

```diff
@@ -1,35 +1,24 @@
-from PyQt4 import QtGui
-from Orange.data import Table
-from Orange.regression.random_forest import (RandomForestRegressionLearner,
-                                             RandomForestRegressor)
-from Orange.widgets.classify.owrandomforest import OWRandomForest
+from Orange.regression.random_forest import RandomForestRegressionLearner
+from Orange.widgets.classify import owrandomforest
 from Orange.widgets import settings
 
 
-class OWRandomForestRegression(OWRandomForest):
+class OWRandomForestRegression(owrandomforest.OWRandomForest):
     name = "Random Forest Regression"
     description = "Random forest regression algorithm."
+    icon = "icons/RandomForest.svg"
+    priority = 40
 
     LEARNER = RandomForestRegressionLearner
-    outputs = [("Learner", LEARNER),
-               ("Model", RandomForestRegressor)]
+    OUTPUT_MODEL_NAME = "Model"
```

Comment:
For other regressors above, it's called 'Predictor', here a 'Model'. We could unify these a bit more too. How about introducing `OWBaseClassifier` and `OWBaseRegressor` that set appropriate output model names?

Compatible with established terminology, we could also just go with 'Model' everywhere. @astaric, @janezd what do you think?